src/services/htmlCompletion.ts

```diff
@@ -442,3 +470,7 @@ function getWordEnd(s: string, offset: number, limit: number): number {
 	}
 	return offset;
 }
+
+function extractAttributeValue(input: string): string {
+	return input.replace(/^['"]/, '').replace(/['"]$/, '');
```

Comment:
Add `todo` for decoding

Reply:
Umm...Initially this function is in `pathCompletion` in html server, so I did not decode since I felt no path completion could happen on a URI.

But here if I decode it (do you mean using `decodeURIComponent(input)`?), I don't know if this would be unwanted for other participants when they are participating on non-URI html attribute values.